testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeAlluxioCaching.java

```diff
@@ -84,6 +94,8 @@ private void testReadFromTable(String tableNameSuffix)
     private void createTestTable(String tableName)
     {
         onTrino().executeQuery("DROP TABLE IF EXISTS " + tableName);
+        onTrino().executeQuery("DROP SCHEMA IF EXISTS delta.test_caching");
```

Comment:
Would be nice to make sure the schema here is the same as the one used in `tableName`.